docs/tutorial/avax-nft.md

```diff
@@ -0,0 +1,399 @@
+
+```shell
+npm install hardhat ethers @nomiclabs/hardhat-ethers --save-dev
+```
+Then, you can check if the installation was successful by typing `npx hardhat --version`. The Hard Hat CLI should print out the version number (yours may be different):
+
+```shell
+npx hardhat --version
+> 2.6.1
+```
+
+Now, with the local simulator nodes still running, run the following commands:
+
+```shell
+npx hardhat accounts --network local
+> 0x8db97C7cEcE249c2b98bDC0226Cc4C2A57BF52FC
+> 0x9632a79656af553F58738B0FB750320158495942
+> 0x55ee05dF718f1a5C1441e76190EB1a19eE2C9430
+> ...
+```
+
+Hopefully you should see a few addresses printed. Next, check the balances with:
+
+```shell
+npx hardhat balances --network local
+> 0x8db97C7cEcE249c2b98bDC0226Cc4C2A57BF52FC has balance 50000000000000000000000000
+> 0x9632a79656af553F58738B0FB750320158495942 has balance 0
+> 0x55ee05dF718f1a5C1441e76190EB1a19eE2C9430 has balance 0
+> ...
+```
+
+If you had correctly [created a keystore user and added test fund](#create-a-keystore-user-and-add-test-fund), you should see one of wealthy addresses shown.
+
+
+## Develop an NFT smart contract
+
+If you already have an existing smart contract, you may skip this section.
+
+We will create [ERC721](https://eips.ethereum.org/EIPS/eip-721) non-fungible tokens with their own attributes. To keep this simple, any account will be able to call a method `mintTo` to mint items.
+
+We will be using the standard [ERC721](https://docs.openzeppelin.com/contracts/4.x/erc721) smart contract from Open Zeppelin. Install it in your project with `npm install @openzeppelin/contract`.
+
+Create a directory named `/contracts` within the project root. Create a smart contract file named `Filet.sol` with the following code:
+
+```js
+// contracts/Filet.sol
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+import "@openzeppelin/contracts/token/ERC721/extensions/ERC721URIStorage.sol";
+import "@openzeppelin/contracts/utils/Counters.sol";
+
+contract Filet is ERC721URIStorage {
+    using Counters for Counters.Counter;
+    Counters.Counter private _tokenIds;
+
+    constructor() ERC721("Filet", "FIT") {}
+
+    function mintTo(address player, string memory tokenURI)
+        public
+        returns (uint256)
+    {
+        _tokenIds.increment();
+        uint256 newItemId = _tokenIds.current();
+        _mint(player, newItemId);
+        _setTokenURI(newItemId, tokenURI);
+
+        return newItemId;
+    }
+}
+
+```
+
+You can come up with your own contract's name, token's name, and token's ticker. I'm naming mine `Filet` here.
+
+Now compile the contract `Filet.sol` using this hardhat command:
+
+```shell
+npx hardhat compile
+> Compiling 1 file with 0.8.0
+> Compilation finished successfully
+```
+
+Once the contract is compiled, create another directory named `script` and add `deploy.js` with the following code:
```

Comment:
*You can come up with your own contract's name, token's name, and token's ticker
At the root level or where? :)